projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts

```diff
@@ -122,10 +122,10 @@ export function registerAndLogin(): void {
 }
 
 export function addProduct(): void {
-  const categoryPageUrl = '/Open-Catalogue/Cameras/Digital-Cameras/c/575';
+  const productPageUrl = '/product/779841';
```

Comment:
Have you checked changing this url does not break any other e2e tests?

Reply:
all good